projects/ui/src/lib/components/po-field/po-decimal/po-decimal.component.html

```diff
@@ -27,5 +27,5 @@
     </div>
   </div>
 
-  <po-field-container-bottom> </po-field-container-bottom>
+  <po-field-container-bottom [p-error-pattern]="getErrorPattern()"> </po-field-container-bottom>
```

Comment:
Compiling TypeScript sources through ngc
ERROR: projects/ui/src/lib/components/po-field/po-decimal/po-decimal.component.html(30,30): Property 'getErrorPattern' does not exist on type 'PoDecimalComponent'.

An unhandled exception occurred: projects/ui/src/lib/components/po-field/po-decimal/po-decimal.component.html(30,30): Property 'getErrorPattern' does not exist on type 'PoDecimalComponent'.

See "/private/var/folders/bs/mnwnbzx546q0nnl4xzhn5m2dg6kqkn/T/ng-o9hfMe/angular-errors.log" for further details.